whois-nrtm/src/main/java/net/ripe/db/whois/nrtm/BaseNrtmServerChannelInitializer.java

```diff
@@ -0,0 +1,61 @@
+package net.ripe.db.whois.nrtm;
+
+import io.netty.buffer.ByteBuf;
+import io.netty.buffer.Unpooled;
+import io.netty.channel.Channel;
+import io.netty.channel.ChannelInitializer;
+import io.netty.channel.ChannelPipeline;
+import io.netty.handler.codec.DelimiterBasedFrameDecoder;
+import io.netty.handler.codec.string.StringDecoder;
+import io.netty.handler.codec.string.StringEncoder;
+import net.ripe.db.whois.common.pipeline.MaintenanceHandler;
+
+import java.nio.charset.StandardCharsets;
+
+
+abstract class BaseNrtmServerChannelInitializer extends ChannelInitializer<Channel> {
```

Comment:
Why use a base class? There's only one implementation

Reply:
This is a renaming of what existed before. As can be seen [here](https://github.com/RIPE-NCC/whois/pull/754/files#diff-71ea4bdce78614e90585c2c7ff267a4d5bb5adebe931de12c203f49fa41485fc). Not sure why the git diff shows it as new distinct addition. 

In the thick of the upgrading, I did not pay attention to such improvement as not using a base case class...but now that you surfaced this, I will look into not having a base class.

Reply:
Removed `BaseNrtmServerChannelInitializer`